browser/resources/settings/brave_rewards_page/brave_rewards_page.js

```diff
@@ -254,6 +195,93 @@ class SettingsBraveRewardsPage extends SettingsBraveRewardsPageBase {
     e.stopPropagation();
     window.open('chrome://restart', '_self');
   }
+
+  adsSubdivisionTargetingCodes_() {
+    if (this.countryCode_ == 'US') {
+      return this.adsUSASubdivisionTargetingCodes_()
+    }
+
+    if (this.countryCode_ == 'CA') {
+      return this.adsCanadaSubdivisionTargetingCodes_()
+    }
+
+    return [
+      { name: loadTimeData.getString('braveRewardsAutoDetectedItem'), value: 'AUTO' },
```

Comment:
Does autodetect make sense in this scenario? I thought it was only relevant when there was a supported subdivision, but not sure.

Reply:
Subdivision targeting should only be shown for US and CA system locales. And only if these are shown should the user be able to choose to auto-detect or select a different region.

Reply:
Agreed, I will change to return only `DISABLED` here.